server/player/playerdb/leveldb.go

```diff
@@ -0,0 +1,58 @@
+package playerdb
+
+import (
+	"encoding/json"
+	"github.com/df-mc/dragonfly/server/player"
+	"github.com/df-mc/goleveldb/leveldb"
+	"github.com/google/uuid"
+	"os"
+)
+
+// Provider ...
```

Comment:
Please document this! Just describe how data is stored (leveldb, json data etc.)